documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc

```diff
@@ -128,6 +128,9 @@ If you do not specify a value, the connector runs an incremental snapshot.
 | An optional string, which specifies a condition based on the column(s) of the {data-collection}(s), to capture a
 subset of the contents of the {data-collection}s.
 For more information about the `additional-condition` parameter, see xref:{context}-incremental-snapshots-additional-condition[].
+|7
+|`surrogate-key`
+|An optional string, which specifies a column name to use as the primary key for the snapshot. This allows to override the default primary key of the table.
```

Comment:
```suggestion
|An optional string that specifies the column name that the connector uses as the primary key for snapshots. 
The specified value overrides the default primary key of the table.
```
I think that you meant to pair this `surrogate-key` callout description with a corresponding `surrogate-key` field with the label `7` in the SQL-based snapshot example  (L63), no?

Reply:
My bad.